### PR TITLE
Add quality metrics

### PR DESCRIPTION
**Pull request by kabilar**
- [x] Update readme to follow new format across all Elements
- [x] Update docs to follow new format across all Elements
  - [x] Add pipeline, partnerships, and roadmap pages
  - [x] Remove Google Analytics Key
  - [x] Remove previous notebooks which are soon to be replaced
- [x] #47 
  - [x] Add `ScanQualityMetrics` table
  - [x] Add `ProcessingQualityMetrics` table to all `imaging*` modules
- [x] Update images with quality metrics tables
- [x] Update version and changelog

### CODE REVIEW

**Review by dimitri-yatsenko — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,38 @@
 # DataJoint Element - Functional Calcium Imaging
 
-DataJoint Element for functional calcium imaging with `ScanImage`, `Scanbox`,
-`Nikon NIS`, or `PrairieView` acquisition software and `Suite2p`, `CaImAn`, and
-`EXTRACT` analysis software. DataJoint Elements collectively standardize and automate
+DataJoint Element for functional calcium imaging with 
+[ScanImage](https://docs.scanimage.org/), 
+[Scanbox](https://scanbox.org/),
+[Nikon NIS-Elements](https://www.microscope.healthcare.nikon.com/products/software/nis-elements), 
+and `Bruker Prairie View` acquisition software; and
```
Comment (kushalbakshi) [inline README.md:7]:
```suggestion
and `Bruker PrairieView` acquisition software; and 
```

File: docs/src/concepts.md [hunk 2]
```diff
@@ -69,133 +55,8 @@ segmented.
...
-guidance from the Scientific Steering Group we will add the following features to this
-Element:
-
-- Data quality metrics
-- Data compression
-- Deepinterpolation
-- Data export to NWB
-- Data publishing to DANDI
+- Nikon NIS-Elements
+- Bruker Prairie View
```
Comment (kushalbakshi) [inline docs/src/concepts.md:62]:
```suggestion
- Bruker PrairieView
```

File: docs/src/index.md [hunk 3]
```diff
@@ -1,14 +1,40 @@
 # Element Calcium Imaging
 
-DataJoint Element for functional calcium imaging with `ScanImage`, `Scanbox`,
-`Nikon NIS`, or `PrairieView` acquisition software and `Suite2p`, `CaImAn`, and
-`EXTRACT` analysis software. DataJoint Elements collectively standardize and automate
+DataJoint Element for functional calcium imaging with 
+[ScanImage](https://docs.scanimage.org/){:target="_blank"}, 
+[Scanbox](https://scanbox.org/){:target="_blank"},
+[Nikon NIS-Elements](https://www.microscope.healthcare.nikon.com/products/software/nis-elements){:target="_blank"}, 
+and `Bruker Prairie View` acquisition software; and
```
Comment (kushalbakshi) [inline docs/src/index.md:7]:
```suggestion
and `Bruker PrairieView` acquisition software; and 
```

File: docs/src/tutorials/index.md [hunk 4]
```diff
@@ -1,64 +1,29 @@
...
+## Videos
 
+The [YouTube tutorial](https://www.youtube.com/watch?v=gFLn0GB1L30) gives an overview of
+the workflow files and notebooks, as well as core concepts related to calcium imaging
+analysis.
 
 ## EXTRACT
 
 Analysis with the EXTRACT package is currently supported for single channel, single
 plane scans with using Suite2p for motion correction. For processing with EXTRACT,
```
Comment (kushalbakshi) [inline docs/src/tutorials/index.md:25]:
```suggestion
plane scans using Suite2p for motion correction. For processing with EXTRACT,
```

**Review by kushalbakshi — COMMENTED** [review]
Thank you, @kabilar. Looks good! Just a few minor things that I spotted but once these are committed, it's ready to merge.

File: README.md (hunk 1 above)
Comment (kabilar) [reply README.md:7]: Thanks @kushalbakshi.  There might be a more accurate source, but the Bruker website lists it as `Prairie View`:

image

Comment (kushalbakshi) [reply README.md:7]: You're correct! Sorry about that. I thought it was one word.

**Review by kushalbakshi — APPROVED** [review]

**Comment by kabilar** [comment]
Version 0.6.0 is tagged, released on GitHub, and published on PyPI.

**Pull request merged**